moc_sprint_tools/create_sprint_boards.py

```diff
@@ -0,0 +1,106 @@
+import click
+import csv
+import datetime
+import github
+import logging
+
+from moc_sprint_tools.sprintman import BoardNotFoundError
+
+LOG = logging.getLogger(__name__)
+
+CARD_COPYING_MAP = {
+    'notes': 'notes',
+    'backlog': 'backlog',
+    'in progress': 'in progress'
+}
+
+
+def load_sprint_data(input_file):
+    sprints = []
+    with open(input_file) as f:
+        reader = csv.reader(f)
+        next(f)
+        for row in reader:
+            sprints.append(
+                [row[0], datetime.datetime.strptime(row[1], '%Y-%m-%d')])
+    return sprints
+
+
+def get_column(board, name):
```

Comment:
Should this be in `sprintman.py`?

Reply:
Perhaps, but if it's part of the `Sprintman` class then I'd have to pass a reference to the instance of the class too. We can refactor this later.